Prepare for `lxc-to-lxd` to not be shipped in the snap (#12951)
canonical/lxd: doc/howto/migrate_from_lxc.md

## Get the tool

> To get the `lxc-to-lxd` tool you can either download a pre-built binary:

Reviewer: ```suggestion
To get the `lxc-to-lxd` tool, you can download a pre-built binary:
```

Not a fan of either/or with so much text in between. ;) But it should work fine without.

1. Download the `bin.linux.lxc-to-lxd` tool ([`bin.linux.lxc-to-lxd.aarch64`](https://github.com/canonical/lxd/releases/latest/download/bin.linux.lxc-to-lxd.aarch64) or [`bin.linux.lxc-to-lxd.x86_64`](https://github.com/canonical/lxd/releases/latest/download/bin.linux.lxc-to-lxd.x86_64)) from the **Assets** section of the latest [LXD release](https://github.com/canonical/lxd/releases).

> 1. Save the binary as `lxc-to-lxd` and make it executable (usually by running `chmod u+x bin.linux.lxc-to-lxd`).

Reviewer: ```suggestion
1. Save the binary as `lxc-to-lxd` and make it executable (usually by running `chmod u+x lxc-to-lxd`).
```

> Or if you have `go` ({ref}`requirements-go`) installed you can get the tool with the following command:

Reviewer: ```suggestion
If you have `go` ({ref}`requirements-go`) installed, you can build the tool with the following command:
```
